tests/toolbox_image.sh

```diff
@@ -13,6 +24,55 @@ result=0
 
 exit_on_failure=false
 
+format_dummy_partition()
+{
+    command="${1}"
+    sector_start="${2}"
+    device_file="${3}"
```

Comment:
device_file == STORAGE_DEVICE_FILE? do we forsee (YAGNI) that we need this parameter variable? It's a 'constant' throughout the program already anyway ...